### PR TITLE
Update to axum 0.7

### PR DESCRIPTION
**Pull request by rakshith-ravi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 # Unreleased
 
+None
+
+# 0.6.0 (05. December, 2023)
+
+- Update to axum 0.7.
+
+# 0.5.0 (15. April, 2023)
+
```
Comment (davidpdrsn) [inline CHANGELOG.md]: This is not released yet so:

```suggestion
- Update to axum 0.7 ([#6])

[#6]: https://github.com/davidpdrsn/axum-typed-websockets/pull/6

# 0.5.0 (15. April, 2023)

```

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "axum-typed-websockets"
-version = "0.4.0"
+version = "0.6.0"
```
Comment (davidpdrsn) [inline Cargo.toml]: Please don't bump this. I'll do that when I make a release.

Not sure why it says 0.4 when the published version is 0.5. I suppose I forgot to push some commits.

**Review by davidpdrsn — CHANGES_REQUESTED** [review]
Thanks!

File: Cargo.toml (hunk 2 above)
Comment (rakshith-ravi) [reply Cargo.toml]: Sure thing. I'll revert it

Comment (rakshith-ravi) [reply Cargo.toml]: Done. Did a rebase to remove it from history as well

**Comment by alekangelov** [comment]
hey @davidpdrsn is there anything else that needs to be done? More than happy to help/chip in it get through to the finish line cheers

**Review by davidpdrsn — APPROVED** [review]

**Comment by davidpdrsn** [comment]
Sorry for the delay! I just published this in 0.6.0 🎉

**Pull request merged**